pfcpiface/messages-conn.go

```diff
@@ -15,6 +14,21 @@ var errFlowDescAbsent = errors.New("flow description not present")
 var errFastpathDown = errors.New("fastpath down")
 var errReqRejected = errors.New("request rejected")
 
+func (pConn *PFCPConn) sendHeartBeatRequest() {
+	seq := pConn.getSeqNum()
+
+	hbreq := message.NewHeartbeatRequest(
+		seq,
+		ie.NewRecoveryTimeStamp(pConn.upf.recoveryTime),
+		nil,
```

Comment:
should we send the source IP IE or nodeIE

Reply:
The spec says that, "This IE may be included when a Network Address Translation device is deployed in the network."
Please let us know, if we need to include this.

Reply:
cc @badhrinathpa